Tests/Model/ModelManagerTest.php

```diff
@@ -403,6 +404,84 @@ public function testAssociationIdentifierType()
         $this->assertSame(42, $result[0]);
     }
 
+    public function getSortableInDataSourceIteratorDataProvider()
+    {
+        return array(
+            array(
+                $sortBy = null,
```

Comment:
IMO the variables are not needed.

@greg0ire should we remove them or use keys instead?